### PR TITLE
Add blog post about upcoming SSL features

### PR DESCRIPTION
**Pull request by SoniaZaldana**
Blog post providing overview about upcoming SSL features including: 
https://issues.redhat.com/browse/ELY-2033
https://issues.redhat.com/browse/ELY-2033

### CODE REVIEW

**Comment by SoniaZaldana** [comment]
@fjuma I updated the blog post based on your feedback :)

**Comment by fjuma** [comment]
Thanks @SoniaZaldana, looks good.

File: _posts/2021-03-19-new-ssl-features.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Upcoming SSL features: Multiple certificate revocation lists support and SSLv2Hello support"
+date: 2021-03-19
+tags: wildfly elytron
+synopsis: An overview of the upcoming SSL/TLS features including the ability to configure
+multiple certificate revocation lists and SSLv2Hello support.
```
Comment (fjuma) [inline _posts/2021-03-19-new-ssl-features.adoc:7]: Just tried checking what the blog post looks like and the multi-line synopsis and the colon in the title cause some issues when rendering. I need to create an aggregate PR to fix some dates in other posts so will fix these issues there.

**Comment by fjuma** [comment]
Superseded by https://github.com/wildfly-security/wildfly-elytron/pull/1503.

**Pull request closed without merging**